.github/workflows/npm-audit-fix.yml

```diff
@@ -0,0 +1,27 @@
+name: npm audit fix
+
+on:
+  schedule:
```

Comment:
I think we decided to remove this workflow from projects, due to the number of PRs?

Reply:
That's right, I'll remove it now 👍